src/Writer/Extension/GooglePlayPodcast/Feed.php

```diff
@@ -67,11 +67,13 @@ public function getEncoding()
     /**
      * Set a block value of "yes" or "no". You may also set an empty string.
      *
-     * @param  string
-     * @return $this
+     * @param string
+     *
+     * @return self
+     *
      * @throws Writer\Exception\InvalidArgumentException
      */
-    public function setPlayPodcastBlock($value)
+    public function setPlayPodcastBlock($value): self
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is final.